server/controllers/cartItems-controller.js

```diff
@@ -35,6 +35,7 @@ const createCartItem = async (req, res) => {
             product_uuid: req.body.product_uuid,
             price: req.body.price,
             image_url: req.body.image_url,
+            place_in_cart: req.body.place_in_cart,
```

Comment:
See above comment about place_in_cart
I see you did a lot of work on it BUT... its not a great pattern.
Can you think of another field to sort by?
Without adding an extra field. Keep things always as simple as needed.